Documentation/guides/port.rst

```diff
@@ -0,0 +1,62 @@
+.. include:: /substitutions.rst
+===========================================
+How to port
+===========================================
+
+This guide explains "How to port NuttX".  At this guide, the scope of porting is to add new SoC/Board directory, the goal of porting is to comfirm to boot NuttShell(NSH).
```

Comment:
please break lines where possible to 80-90 characters so that documentation can be read better in a split-screen terminal, like for example in https://raw.githubusercontent.com/apache/nuttx/refs/heads/master/Documentation/guides/ipv6.rst